pkgs/development/python-modules/pyvisa-sim/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, fetchPypi
+, pythonOlder
+, buildPythonPackage
+, pyvisa
+, pyyaml
+, stringparser
+, typing-extensions
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "PyVISA-sim";
```

Comment:
```suggestion
  pname = "pyvisa-sim";
```

Reply:
Ok, then I'll have to set `pname` in `fetchPypi` independently I guess, as it seems to be case sensitive. wilco